tests/_data/media/logo.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is a 680 byte file used as test data. What's the value in minifying it?

You are correct, though, that we should also test with a zipped version.

Reply:
Fair, it is small, but it could still be [~64% smaller](http://p.tri.be/al96ej). Uploaded the compressed version in Slack. Github doesn't like it.